libs/BLEKit/include/BLEGap.h

```diff
@@ -0,0 +1,69 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _LEKA_OS_LIB_BLE_GAP_H_
+#define _LEKA_OS_LIB_BLE_GAP_H_
+
+#include "events/mbed_events.h"
+
+#include "ble/BLE.h"
+#include "ble/Gap.h"
+
+#include "cxxsupport/lstd_span"
+
+namespace leka {
+
+static constexpr uint16_t MAX_ADVERTISING_PAYLOAD_SIZE = 50;
```

Comment:
le static ne sert à rien ici

```suggestion
constexpr uint16_t MAX_ADVERTISING_PAYLOAD_SIZE = 50;
```